README.md

```diff
@@ -15,6 +15,10 @@ KubeRay provides several tools to improve running and managing Ray's experience
 - Kubernetes event dumper for ray clusters/pod/services (future work)
 - Operator Integration with Kubernetes node problem detector (future work)
 
+## Documentation
+
+You can view detailed documentation and guides at [https://github.com/ray-project/kuberay](https://github.com/ray-project/kuberay)
```

Comment:
Do you like to link to `https://ray-project.github.io/kuberay/`? or just github home page?

Reply:
opps nice catch!